### DISCUSSION TITLE
KillCurly not disabling GoGuardian?

### DISCUSSION THREAD

**Discussion by Ianrid1** [3kh0/ext-remover#476] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Iv been using KillCurly for a week but i just started to realise everytime i use it it doesnt disable go guardian. Please help!

**Comment by Aka-but-nice** [comment, marked as answer]
Are sites still blocked?

**Reply by Ianrid1** [reply]
No
